### PR TITLE
Add MIGS payment gateway

### PR DESCRIPTION
**Pull request by jacobcat**
This branch adds MasterCard Internet Gateway Service, and briefly modifies the EventCheckoutController class to support different redirection types (POST and GET).

### CODE REVIEW

**Comment by daveearley** [comment]
Cheers for this @jacobcat ! I'll have give it a test soon.

File: resources/views/Public/ViewEvent/Embedded/EventPageCheckout.blade.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @extends('Public.ViewEvent.Layouts.EmbeddedEventPage')
 
 @section('head')
-<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
+<!-- script type="text/javascript" src="https://js.stripe.com/v2/"></script -->
```
Comment (daveearley) [inline resources/views/Public/ViewEvent/Embedded/EventPageCheckout.blade.php]: We need to keep this uncommitted by default so people Stripe will work.

File: resources/views/Public/ViewEvent/EventPageCheckout.blade.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @extends('Public.ViewEvent.Layouts.EventPage')
 
 @section('head')
-    <script type="text/javascript" src="https://js.stripe.com/v2/"></script>
+    <!-- script type="text/javascript" src="https://js.stripe.com/v2/"></script -->
```
Comment (daveearley) [inline resources/views/Public/ViewEvent/EventPageCheckout.blade.php]: Same as above

File: config/attendize.php [hunk 3]
```diff
@@ -54,14 +55,14 @@
...
-    'default_currency'           => 2, #Euro
+    'default_timezone'           => 92, #Asia/Singapore
+    'default_currency'           => 15, #Philippine Peso
     'default_date_format'        => 'j M, Y',
     'default_date_picker_format' => 'd M, yyyy',
     'default_datetime_format'    => 'F j, Y, g:i a',
     'default_query_cache'        => 120, #Minutes
     'default_locale'             => 'en',
-    'default_payment_gateway'    => 1, #Stripe=1 Paypal=2 BitPay=3
+    'default_payment_gateway'    => 4, #Stripe=1 Paypal=2 BitPay=3 MIGS=4
```
Comment (daveearley) [inline config/attendize.php]: It's best to leave this default as Stripe as that is what the majority of people use. Same above for the currency.

Comment (jacobcat) [inline config/attendize.php]: Sure, no problem. Will return them to the defaults. Thanks!

File: resources/views/Public/ViewEvent/Embedded/EventPageCheckout.blade.php (hunk 1 above)
Comment (jacobcat) [inline resources/views/Public/ViewEvent/Embedded/EventPageCheckout.blade.php]: Oh, yeah, I didn't intend to disable that; I just debugged it and temporarily commented it--will put it back ;-)

**Comment by jacobcat** [comment]
Hi @daveearley,

I've done your requested changes... please check again. Thanks! :-)

**Comment by daveearley** [comment]
Cheers @jacobcat!

**Pull request merged**